### PR TITLE
added new demos

### PR DESCRIPTION
**Pull request by maciekswat**
added new demos for the new modules

### CODE REVIEW

File: CompuCell3D/core/Demos/SteppableDemos/PolygonInitializer/Simulation/cellsort_2D.xml [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+			   <To x="2" y="25" />
+			</Edge>
+			<Edge>
+			   <From x="2" y="25" />
+			   <To x="20" y="40" />
+			</Edge>
+		 </EdgeList>
+      </Region>
+   </Steppable-->
+git fetch fork
```
Comment (fyffep) [inline CompuCell3D/core/Demos/SteppableDemos/PolygonInitializer/Simulation/cellsort_2D.xml:171]: Remove line

File: CompuCell3D/core/Demos/SteppableDemos/TubeInitializer/Simulation/cellsort_2D.xml [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    </Plugin>
+
+
+<!--Case 4: Thick two-layer tube-->
+   <Steppable Type="TubeInitializer">
+      <Region>
+	 	 <InnerRadius>8</InnerRadius>
+	 	 <OuterRadius>14</OuterRadius>
+         <Gap>0</Gap>
+         <Width>2</Width>
```
Comment (fyffep) [inline CompuCell3D/core/Demos/SteppableDemos/TubeInitializer/Simulation/cellsort_2D.xml:40]: Add the tag ``<NumSlices>8</NumSlices>`` inside ``Region``.

File: CompuCell3D/core/Demos/SteppableDemos/TubeInitializer/Simulation/cellsort_2D.xml [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+		 <Extrude>
+		   <From x="15" y="45" z="25" />
+		   <To x="15" y="10" z="25" />
+		 </Extrude>
+      </Region>
+      <Region>
+	 	 <InnerRadius>3</InnerRadius>
+	 	 <OuterRadius>8</OuterRadius>
+         <Gap>0</Gap>
+         <Width>2</Width>
```
Comment (fyffep) [inline CompuCell3D/core/Demos/SteppableDemos/TubeInitializer/Simulation/cellsort_2D.xml:51]: Add the tag ``<NumSlices>10</NumSlices>`` inside ``Region``.

File: CompuCell3D/core/Demos/SteppableDemos/PolygonInitializer/Simulation/cellsort_2D.xml [hunk 4]
```diff
@@ -0,0 +1,208 @@
+<CompuCell3D>
```
Comment (fyffep) [inline CompuCell3D/core/Demos/SteppableDemos/PolygonInitializer/Simulation/cellsort_2D.xml:1]: Do you think we should add a comment at the top of each one like this?
``<!--See the documentation for more details: https://compucell3dreferencemanual.readthedocs.io/en/latest/polygon_initializer.html -->``
And, similarly, for tubes...
``<!--See the documentation for more details: https://compucell3dreferencemanual.readthedocs.io/en/latest/tube_initializer.html -->``

**Review by fyffep — CHANGES_REQUESTED** [review]

**Comment by maciekswat** [comment]
I made the fixes and added comments - those are indeed useful

**Comment by fyffep** [comment]
> I made the fixes and added comments - those are indeed useful

Great, I see the changes. ``NumSlices`` isn't there, but it's also an optional tag and described in the documentation anyway.

**Comment by maciekswat** [comment]
oops I forgot to add it. We can make a note and add it next time

**Pull request merged**